content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -293,6 +293,78 @@ Weave Net runs as a [CNI plug-in](https://www.weave.works/docs/net/latest/cni-pl
+{{< warning >}}Kubernetes' implementation of dual-stack networking is in flux. You may well find that the design is changed in future releases.{{< /warning >}}
+
+IPv4/IPv6 dual stack enabled Kubernetes clusters allow the assignment of both IPv4 and IPv6 addresses simultaneously which enables the following capabilities:
+
+   * Awareness of multiple IPv4/IPv6 address assignments per Pod
+   * Native IPv4-to-IPv4 in parallel with IPv6-to-IPv6 communications to, from, and within a cluster
+
+Details on the implementation of this feature may be found in the [IPv4/IPv6 dual stack KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/20180612-ipv4-ipv6-dual-stack.md)
+
+### Supported Features
+
+Enabling IPv4/IPv6 dual stack on a Kubernetes cluster enables the following features:
+
+   * Dual stack Pod networking (multi-IP Pod)
+   * Kubenet multi address family support (IPv4 and IPv6)
+   * Egress Pod Internet routing via IPv4/IPv6 addresses
+
+### Prerequisites
+
+The following prerequisites are needed in order to utilize IPv4/IPv6 dual stack Kubernetes clusters:
+
+   * Provider support for dual stack networking (Cloud provider or otherwise must be able to provide Kubernetes nodes with routable IPv4/IPv6 network interfaces)
+   * Kubenet network plugin
+
+### Enable IPv4/IPv6 dual stack
+
+To enable IPv4/IPv6 dual stack, the following flags must added to these cluster components:
+
+   * kube-controller-manager:
+      * `--feature-gates="IPv6DualStack=true"`
+      * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>` eg. `--cluster-cidr=10.244.0.0/16,fc00::/24`
+   * kubelet:
+      * `--feature-gates="IPv6DualStack=true"`
+
+### Validate IPv4/IPv6 dual stack
+
+Validate IPv4/IPv6 Pod CIDRs have been assigned on a dual stack enabled node by running the following command. There should be a single IPv4 and IPv6 CIDR allocated (replace node name with a valid node from the cluster. In this example the node name is k8s-linuxpool1-34450317-0):
+
+```shell
+kubectl get nodes k8s-linuxpool1-34450317-0 -o go-template --template='{{range .spec.podCIDRs}}{{printf "%s\n" .}}{{end}}'
+10.244.1.0/24
+a00:100::/24
+```
+There should be a single IPv4 and IPv6 CIDR allocated
+
+
+Validate that the node has an IPv4 and IPv6 interface detected (replace node name with a valid node from the cluster. In this example the node name is k8s-linuxpool1-34450317-0): 
+```shell
+kubectl get nodes k8s-linuxpool1-34450317-0 -o go-template --template='{{range .status.addresses}}{{printf "%s: %s \n" .type .address}}{{end}}'
+Hostname: k8s-linuxpool1-34450317-0
+InternalIP: 10.240.0.5
+InternalIP: 2001:1234:5678:9abc::5
+```
+
+Validate that a Pod has an IPv4 and IPv6 address assigned. (replace Pod name with a valid Pod on the cluster. In this example the node name is n01-79845568c8-jm9sq)
+```shell
+kubectl get pods n01-79845568c8-jm9sq -o go-template --template='{{range .status.podIPs}}{{printf "%s \n" .ip}}{{end}}'
+10.244.1.4
+a00:100::4
+```
+
+### Known Issues
```

Comment:
The style guide [recommends](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future) avoiding statements that are tied to a point in time. That's especially relevant to Concept pages; ideally, the _concept_ information can stay the same even as the details change.

It is worth mentioning the /24 default size for on IPv6 CIDR blocks; this could surprise unwary readers. I recommend using a `{{< note >}}` shortcode and placing that higher on the page.

The details about kubenet are also important. Maybe under a “Kubenet” heading?